### PR TITLE
Add missing symbol define

### PR DESCRIPTION
**Pull request by ethercflow**
Signed-off-by: Wenbo Zhang <ethercflow@gmail.com>

### CODE REVIEW

File: librocksdb_sys/bindings/aarch64-unknown-linux-gnu-bindings.rs [hunk 1]
```diff
@@ -2260,6 +2260,11 @@ extern "C" {
         v: libc::c_uchar,
     );
 }
+extern "C" {
+    pub fn crocksdb_options_is_enable_multi_batch_write(
+        opt: *mut crocksdb_options_t,
+    ) -> libc::c_uchar;
+}
```
Comment (tabokie) [inline librocksdb_sys/bindings/aarch64-unknown-linux-gnu-bindings.rs:2267]: It's not used. No need to update.

**Review by tabokie — APPROVED** [review]

File: librocksdb_sys/bindings/aarch64-unknown-linux-gnu-bindings.rs (hunk 1 above)
Comment (ethercflow) [reply librocksdb_sys/bindings/aarch64-unknown-linux-gnu-bindings.rs]: Removed, thanks!

**Comment by tabokie** [comment]
Could you make sure it's ready for TiKV to use?

**Comment by ethercflow** [comment]
> Could you make sure it's ready for TiKV to use?

At present, it has been successfully compiled and linked by TiKV, I'll built a single-node cluster to test the whole process to make sure no other issues.

**Pull request merged**